README.md

```diff
@@ -0,0 +1,115 @@
+# Badgr
+
+![badgr](https://badgr.brigade2.io/v1/github/checks/99005/brigadecore/badgr/badge.svg)
+[![codecov](https://codecov.io/gh/brigadecore/badgr/branch/main/graph/badge.svg?token=N1SQx2TZt0)](https://codecov.io/gh/brigadecore/badgr)
+[![Go Report Card](https://goreportcard.com/badge/github.com/brigadecore/badgr)](https://goreportcard.com/report/github.com/brigadecore/badgr)
+
+Badgr's creators love using [shields.io](https://shields.io/), for displaying
+various project statuses in our READMEs, but it doesn't support something we
+wanted badly-- the ability to generate a badge based on the results of a GitHub
```

Comment:
This doc focuses in Badgr being utilized for Check Suite badges -- do we also want to add a mention around the possible use for checking branch status with Badger (BuildBadge)?

Reply:
Ya... so about that...

Recall that I started work on this well over a year ago. I can't remember _why_ I included the branch status functionality, so I was debating cutting it. I've held off on doing so, but elected to at least not document it for now.

Reply:
Offline, you and I decided the second endpoint is not needed.